client/components/payment-activity/other-payments-data.tsx

```diff
@@ -0,0 +1,19 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+
+/**
+ * Internal dependencies.
+ */
+import './style.scss';
+
+const OtherPaymentsData: React.FC = () => {
```

Comment:
Did not prefer the name `OtherPaymentsData`. Maybe something like `DataHighlights`as it is referred to in the design?

Reply:
Changed